File: AurigainLoanERPApp/src/app/Shared/Model/master-model/payment-mode.model.ts
export class PaymentModeModel {
  Id!: number;
        Mode!: string | null;
        MinimumValue!: number| null;
        IsActive!: boolean | null;
        IsDelete!: boolean;
        CreatedOn!: Date;
        ModifiedOn!: Date | null;
}

export class DDLPaymentModeModel {
  Id!: number;
  Mode!: string | null;
}

File: AurigainLoanERPApi/AurigainLoanERP.Shared/ContractModel/PaymentModeModel.cs
using System;

namespace AurigainLoanERP.Shared.ContractModel
{
    public class PaymentModeModel
    {
        public int Id { get; set; }
        public string Mode { get; set; }
        public long? MinimumValue { get; set; }
        public bool? IsActive { get; set; }
        public bool IsDelete { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime? ModifiedOn { get; set; }
    }
    /// <summary>
    /// Payment Mode Dropdown Model
    /// </summary>
    public class DDLPaymentModeModel
    {
        public int Id { get; set; }
        public string Mode { get; set; }

    }
}
